ProcessLib/SmallDeformation/CreateSmallDeformationProcess.cpp

```diff
@@ -119,7 +119,10 @@ createSmallDeformationProcess(
                 "There are subdomains being deactivated."
                 "Please make sure that there is no boundary condition "
                 "at any node inside these subdomains, "
-                "and only the iterative linear solver is utilized.");
+                "and only the iterative linear solver is utilized.\n"
+                "If  there are boundary conditions  at the nodes inside these "
```

Comment:
Does `inside` mean "entirely within the deactivated region"?
In other words: Are BCs at the interface between activated and deactivated regions possible?

Reply:
The nodes at the interface  between activated and deactivated regions  are still active, and they are be assigned with BCs.